AEGIS-6406 fix scripts

jamienutter opened the pull request:
Removed the scripts and made them modules.
Tidied the code and added a bunch of #TODO items throughout.

dtpryce reviewed (approved):
I like how you have planned out stuff for others to do via todos. My only comment that doesn't block merging, it might be worth keeping that old README somewhere so you can understand those scripts / class usage until it's ready to be fully updated. I guess it's here in history and in old fork but just a thought.

jamienutter commented:
> I like how you have planned out stuff for others to do via todos. My only comment that doesn't block merging, it might be worth keeping that old README somewhere so you can understand those scripts / class usage until it's ready to be fully updated. I guess it's here in history and in old fork but just a thought.

I've removed the scripts so I also removed from the README as it no longer correlates. 
The scripts and old README are in history and the original code too
